### ISSUE TITLE
kubectl diff segfaults when the object doesn't exist

### ISSUE THREAD

**Issue by smaslennikov** [kubernetes/kubernetes#69450]
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line:
>
/kind bug
> /kind feature

**What happened**:

`kubectl alpha diff -f pizza.yaml` will segfault loudly if the objects within don't exist in K8s already. I hit this by failing to specify the namespace: my objects were not in the default namespace. With `-v 6` I found that a 404 was returned by my API server.

**What you expected to happen**:

User is to be notified that the object(s) don't exist.

**How to reproduce it (as minimally and precisely as possible)**:

1. Have a YAML
2. Don't apply it
3. `kubectl alpha diff -f` your yaml

**Anything else we need to know?**:

Huge thank you for putting together this feature in the first place. It's a big help!

**Environment**:
- Kubernetes version (use `kubectl version`):

```
Client Version: version.Info{Major:"1", Minor:"11", GitVersion:"v1.11.3", GitCommit:"a4529464e4629c21224b3d52edfe0ea91b072862", GitTreeState:"clean", BuildDate:"2018-09-10T11:44:36Z", GoVersion:"go1.11", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"10+", GitVersion:"v1.10.3-eks", GitCommit:"58c199a59046dbf0a13a387d3491a39213be53df", GitTreeState:"clean", BuildDate:"2018-09-21T21:00:04Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"linux/amd64"}
```

- Cloud provider or hardware configuration: EKS
- OS (e.g. from /etc/os-release): OSX 10.13.6
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

**Comment by benmoss** [comment]
This looks like it works in 1.12.1 for me. It gives a diff that is all delta.

**Comment by benmoss** [comment]
Since this is already fixed in 1.12, closing!

/close
